woocommerce-services.php

```diff
@@ -48,7 +48,7 @@
 	define( 'WOOCOMMERCE_CONNECT_MAX_JSON_DECODE_DEPTH', 32 );
 
 	if ( ! defined( 'WOOCOMMERCE_CONNECT_SERVER_API_VERSION ' ) ) {
-		define( 'WOOCOMMERCE_CONNECT_SERVER_API_VERSION', '4');
```

Comment:
New version (introduced in https://github.com/Automattic/woocommerce-connect-server/pull/1649 ) - newer versions of the server will return new carriers, without breaking older versions of the plugin.